src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs

```diff
@@ -51,7 +54,12 @@ public virtual DownloadSpecDecision IsSatisfiedBy(RemoteEpisode subject, SearchC
                 _logger.Debug("Checking current status of episode [{0}] in history", episode.Id);
                 var mostRecent = _historyService.MostRecentForEpisode(episode.Id);
 
-                if (mostRecent != null && mostRecent.EventType == EpisodeHistoryEventType.Grabbed)
+                if (mostRecent == null)
```

Comment:
Instead of only getting the most recent item, why not get all the events for the episode? If the latest is grab then it can handle that like normal. Otherwise you can loop until you find an event type that you're looking for.